PlayerOne/canvas.js

```diff
@@ -5,8 +5,8 @@ const X_AXIS_COMPENSATION = - 670
 window.addEventListener("load", () => {
     const canvas = document.querySelector("#canvas");
     const ctx = canvas.getContext("2d");
-    // canvas.height = window.innerHeight - 500;
-    // canvas.width = window.innerWidth - 500;
+    canvas.height = 500;
+    canvas.width = 600;
```

Comment:
canvas height and width should be defined in javascript directly accessing the DOM, or in the HTML file because this allows the `eventListener` to get the position of the mouse in respect to the viewport more accurately. This is why we were having drawings before that were way off the mouse and looked large and pixelated.